test/verify/check-realms

```diff
@@ -878,10 +875,12 @@ class TestPackageInstall(packagelib.PackageCase):
         b.wait_popdown("realms-op")
 
         # should not have a tooltip any more
-        b.mouse("#system-info-domain span", "mouseover")
-        time.sleep(2)
-        self.assertFalse(b.is_present(".tooltip-inner"))
-
+        if self.machine.image in ["rhel-8-1-distropkg"]:
+            b.mouse("#system-info-domain span", "mouseover")
+            time.sleep(2)
+            self.assertFalse(b.is_present(".tooltip-inner"))
+        else:
+            b.wait_attr("#system_information_domain_tooltip", "data-test-has-tooltip", None)
```

Comment:
That element now doesn't exist any more if there's no tooltip. It should become  `bwait_not_present(`#system_information_domain_tooltip`).

Reply:
Ouch, well spotted.